api/server/structs/endpoints_beacon.go

```diff
@@ -133,6 +133,13 @@ type GetBlockAttestationsResponse struct {
 	Data                []*Attestation `json:"data"`
 }
 
+type GetBlockAttestationsV2Response struct {
+	Version             string         `json:"version"`
+	ExecutionOptimistic bool           `json:"execution_optimistic"`
+	Finalized           bool           `json:"finalized"`
+	Data                []*Attestation `json:"data"`
```

Comment:
I don't think this is correct, it would need to handle both Attestation and AttestationElectra values